app/controllers/VatDetailsController.scala

```diff
@@ -163,17 +160,16 @@ class VatDetailsController @Inject()(val messagesApi: MessagesApi,
     }
   }
 
-  private def retrieveIsNonMTDfB(mandationStatus: HttpGetResult[MandationStatus]): Option[Boolean] = {
+  private[controllers] def retrieveIsOfStatus(mandationStatus: HttpGetResult[MandationStatus], expectedType: String*): Option[Boolean] = {
```

Comment:
Can you change `expectedType` to just be a `Seq[String]` rather than using varargs, unless I'm missing something.

Reply:
Was for a reason when those other functions existed. Doesn't make a difference now. Will remove.